cmplibrary/src/test/java/com/sourcepoint/cmplibrary/consent/ClientEventManagerTest.kt

```diff
@@ -38,118 +35,116 @@ class ClientEventManagerTest {
             spClient = spClient,
             executor = MockExecutorManager()
         )
-    }
-
-    @Test
-    fun `GIVEN 2 successfully sendConsent (GDPR, CCPA) calls, TRIGGER 1 onSpFinish`() {
 
         every { consentManagerUtils.gdprConsentOptimized }.returns(Either.Right(GDPRConsentInternal()))
         every { consentManagerUtils.ccpaConsentOptimized }.returns(Either.Right(CCPAConsentInternal()))
```

Comment:
why did you decide to put those 2 lines into `setup`? do we use those lines in the majority of the test cases?